### PR TITLE
Add net452 target framework support

### PR DESCRIPTION
**Pull request by gabi-cg**
Add support for net framework 452

### CODE REVIEW

File: src/MakingSense.AspNetCore.HypermediaApi/MakingSense.AspNetCore.HypermediaApi.csproj [hunk 1]
```diff
@@ -45,10 +45,13 @@
...
+  <ItemGroup Condition="'$(TargetFramework)' == 'net452'">
+	<PackageReference Include="Microsoft.AspNetCore.Mvc" Version="1.1.2" />
+  </ItemGroup>
+
   <ItemGroup>
     <PackageReference Include="System.Linq" Version="4.3.0" />
     <PackageReference Include="System.Linq.Queryable" Version="4.3.0" />
     <PackageReference Include="System.Threading" Version="4.3.0" />
   </ItemGroup>
-
```
Comment (elsupergomez) [inline src/MakingSense.AspNetCore.HypermediaApi/MakingSense.AspNetCore.HypermediaApi.csproj:53]: Please do not remove this line

File: src/MakingSense.AspNetCore.HypermediaApi/Formatters/HypermediaApiJsonInputFormatter.cs [hunk 2]
```diff
@@ -1,10 +1,10 @@
-﻿using System;
-using System.Buffers;
-using Microsoft.AspNetCore.Mvc;
+﻿using Microsoft.AspNetCore.Mvc;
```
Comment (elsupergomez) [inline src/MakingSense.AspNetCore.HypermediaApi/Formatters/HypermediaApiJsonInputFormatter.cs:1]: Please revert format changes to make them in the scope of another PR.
This comment apply for all file

**Review by elsupergomez — COMMENTED** [review]
Only some minor format changes.

Great work @gabi-cg !!

**Comment by andresmoschini** [comment]
WOOOOOOW! It is great! 🤯🎉 

Good job @gabi-cg and @elsupergomez !!

File: src/MakingSense.AspNetCore.HypermediaApi/MakingSense.AspNetCore.HypermediaApi.csproj [hunk 3]
```diff
@@ -45,10 +45,14 @@
...
+
```
Comment (elsupergomez) [inline src/MakingSense.AspNetCore.HypermediaApi/MakingSense.AspNetCore.HypermediaApi.csproj:57]: @gabi-cg please clean extra whitespaces

File: src/MakingSense.AspNetCore.HypermediaApi/Formatters/HypermediaApiJsonInputFormatter.cs (hunk 2 above)
Comment (elsupergomez) [reply src/MakingSense.AspNetCore.HypermediaApi/Formatters/HypermediaApiJsonInputFormatter.cs:1]: @gabi-cg remember this ;)

**Review by elsupergomez — APPROVED** [review]
🚀 great work @gabi-cg
